Week1/homkework/app.js

```diff
@@ -0,0 +1,76 @@
+const mysql = require('mysql');
+const executeQuery = require('./executeQuery');
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: 'mysql1',
+});
+
+connection.connect();
+
+executeQuery.fnExecuteQuery(connection, 'CREATE DATABASE world');
+executeQuery.fnExecuteQuery(connection, 'use world');
+
+executeQuery.fnExecuteQuery(
+  connection,
+  'create table country(\
+    name VARCHAR(50),\
+    continent VARCHAR(25),\
+    region VARCHAR(50),\
+    surfacearea INT,\
+    indepyear INT,\
+    population INT,\
+    lifeexpectancy INT,\
+    gnp INT,\
+    gnpold INT,\
+    localname VARCHAR(50),\
+    govermentform VARCHAR(50),\
+    headofstate VARCHAR(50),\
+    capital VARCHAR(50)\
+    )',
```

Comment:
Be careful some values here can not be INT. Integer means no floating point, could you find which one and fix them ?